MAUI/PDF-Viewer/Built-In-Toolbar.md

```diff
@@ -0,0 +1,741 @@
+---
+
+# Working with the Built-In Toolbar in .NET MAUI Pdf Viewer (SfPdfViewer)
+
+The [SfPdfViewer](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.PdfViewer.SfPdfViewer.html) has a built-in toolbar, which allows you to perform operations such as adding, modifying annotations, searching text, etc. Additionally, you have the option to show or hide the default built-in toolbar and its sub-toolbars and toolbar items, such as buttons, and you can customize your own toolbar by disabling the built-in toolbar and implementing one at the application and hooking the custom toolbar UI to the PDF viewer APIs. You can see the custom toolbar example [here](https://github.com/SyncfusionExamples/maui-pdf-viewer-examples/tree/master/Custom%20Toolbar).
+
+## Show/hide the built-in toolbar
+
+By default, the built-in toolbar is visible. You can `hide` it by setting the `ShowToolbars` property of the PDF Viewer to `false`. And you can `show` the built-in toolbar by setting the `ShowToolbars` property of the PDF Viewer to `true`.
+
+{% tabs %}
+{% highlight C# %}
+//set the ShowToolbars to false
+public MainPage()
+{
+     SfPdfViewer pdfViewer=new SfPdfViewer();
+     pdfViewer.ShowToolbars = false;
+}  
+{% endhighlight %}
+{% endtabs %}
+
+{% tabs %}
+{% highlight XAML %}
+ <ContentPage.Content>
+
+  <syncfusion:SfPdfViewer x:Name="pdfViewer" DocumentSource ="{Binding PdfDocumentStream} " ShowToolbars="False" />
+
+  </ContentPage.Content>
+{% endhighlight %}
+{% endtabs %}
+
+## Customize the Toolbar 
+
+To customize the toolbar, you can programmatically add, remove, and hide the items based on your needs.
+
+### Customizing toolbar items using the GetByName method
+
+### Accessing and Modifying the Visibility of the Toolbar Items
+
+By using the `GetByName` method with its specified toolbar or icon names, you can access the properties of those toolbars or items. After accessing the item, you can modify its visibility by setting the `IsVisible` property. Here, we retrieve the toolbar "PrimaryToolbar" and "Search" item using the GetByName method and hide the "Search" item by setting its visibility to false.
+
+{% tabs %}
+{% highlight C# %}
+// On the desktop, we access the search button located in the primary toolbar using the GetByName method.
+
+var item = pdfViewer.Toolbars?.GetByName("PrimaryToolbar")?.Items?.GetByName("Search");
+if (item != null)
+{
+item.IsVisible = false; // Hide the search item
+}
+{% endhighlight %}
+{% endtabs %}
+
+### Adding a New Toolbar Item
+
+To add an item to the toolbar in [SfPdfViewer](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.PdfViewer.SfPdfViewer.html), first, create the button you want to include. Then, convert that button into a ToolbarItem using the `ToolbarItem` method. Finally, add the newly created ToolbarItem to the toolbar using the `Add` method. Here we create the new button fileOpenButton and retrieve the "PrimaryToolbar" using the GetByName method. Add the new button to the "Primary Toolbar".
```

Comment:
Generalise the item or say it as an example.

<img width="339" alt="image" src="https://github.com/syncfusion-content/maui-docs/assets/65169077/218b19b6-b07f-4c79-8909-948a55ed7d6b">

In the above contents, it conveys only button can be added as item.